bin/eslint.js

```diff
@@ -209,6 +214,8 @@ function analyzeFiles() {
         var path = result.filePath.replace(/^\/code\//, "");
 
         result.messages.forEach(function(message) {
+          if (ignoreWarnings && message.severity === 1) { return; }
```

Comment:
I'm assuming that `1` maps to the warning level but can we explicitly map this value to a named variable? Maybe up top as `WARNING_LEVEL`?